src/types.ts

```diff
@@ -26,6 +28,8 @@ export interface MojoStash {[key: string]: any}
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export type MojoPlugin = (app: App, options: MojoStash) => any;
 
+export type MojoRenderOptions = ({maybe?: boolean, view?: string, status?: number} | string);
```

Comment:
What is the plain string? And where are `text`, `json` and so on?

Reply:
plain string is https://github.com/mojolicious/mojo.js/blob/main/src/context/http.js#L26 - I guess that render method should actually take `MojoRenderOptions | string` instead.

Reply:
Agreed. 👍